modules/utils.lua

```diff
@@ -147,12 +149,14 @@ if isServer then
         model = type(model) == 'string' and joaat(model) or (model --[[@as integer]])
         local ped = GetPlayerPed(source)
 
-        return qbx.spawnVehicle({
+        local netId, _ = qbx.spawnVehicle({
```

Comment:
```suggestion
        local netId = qbx.spawnVehicle({
```
Can this be omitted? Or is that another diagnostic warning?

Reply:
It can be omitted. I Just do that out of habit to show that it does in fact return two values but we only use one.

Reply:
Makes sense.